tests/model_hub_tests/torch_tests/test_detectron2.py

```diff
@@ -82,6 +85,10 @@ def compare_results(self, fw_outputs, ov_outputs):
             is_ok = compare_two_tensors(cur_ov_res[:l], cur_fw_res[:l], fw_eps)
         assert is_ok, "Accuracy validation failed"
 
+    def teardown_class(self):
+        subprocess.run(
+            ["python3", "-m", "pip", "uninstall", "-y", "detectron2"])
```

Comment:
I think it is better to use sys.executable instead of python3 for avoiding accidental usage wrong env